airbyte-db/lib/src/test/java/io/airbyte/db/check/impl/ConfigsDatabaseAvailabilityCheckTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.check.impl;
+
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.jooq.DSLContext;
+import org.jooq.Select;
+import org.jooq.exception.DataAccessException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test suite for the {@link ConfigsDatabaseAvailabilityCheck} class.
+ */
+public class ConfigsDatabaseAvailabilityCheckTest extends AbstractDatabaseAvailabilityCheckTest {
+
+  @Test
+  void checkDatabaseAvailability() {
+    final var check = new ConfigsDatabaseAvailabilityCheck(dslContext, 60000L);
+    Assertions.assertDoesNotThrow(() -> check.check());
+  }
+
+  @Test
+  void checkDatabaseAvailabilityTimeout() {
+    final DSLContext dslContext = mock(DSLContext.class);
+    when(dslContext.fetchExists(any(Select.class))).thenThrow(new DataAccessException("test"));
+    final var check = new ConfigsDatabaseAvailabilityCheck(dslContext, 2000L);
```

Comment:
Similar question here - it seems like these tests would be much quicker if these values were set to something like `200L` instead of `2000L` without much downside

Reply:
I can try going lower -- I don't want to create false positives because we use too low a value for even the tests to complete.  I will play around with lowering it to see what we can get away with.  The proof will be in the PR build, as those seem to perform a bit worse than on your local machine.